src/core/url-beautifier/search/generator.ts

```diff
@@ -65,80 +109,71 @@ export default class SearchUrlGenerator extends UrlGenerator<UrlBeautifier.Searc
     return path;
   }
 
-  buildUrl(path: string[], query: object) {
-    let url = `/${path.map((part) => encodeURIComponent(part)).join('/')}`;
-
-    if (this.config.suffix) {
-      url += `/${this.config.suffix.replace(/^\/+/, '')}`;
-    }
-
-    const queryPart = Object.keys(query)
-      .sort()
-      .map((key) => `${key}=${encodeURIComponent(query[key])}`)
-      .join('&');
-
-    if (queryPart) {
-      url += '?' + queryPart;
-    }
-
-    return url.replace(/\s|%20/g, '-');
+  static convertSort(sort: Store.Sort) {
+    const { field, descending = !!sort.descending } = sort;
+    return [field, [descending]];
```

Comment:
I might just not know how this should look, but should `descending` be in its own sub-array?